src/js/apps/patients/sidebar/filters_state.js

```diff
@@ -0,0 +1,47 @@
+import { sortBy, isEqual, omit, size, isNull, clone } from 'underscore';
+
+import Backbone from 'backbone';
+
+export default Backbone.Model.extend({
+  getDefaultSelectedStates() {
+    return clone(this.get('defaultStates'));
+  },
+  getFilters() {
+    return clone(this.get('filters'));
+  },
+  getStatesFilters() {
+    return clone(this.get('states'));
+  },
+  getFilter(name) {
+    return this.get('filters')[name];
+  },
+  setFilter(name, value) {
+    const filters = this.getFilters();
+    filters[name] = value;
+    this.set('filters', filters);
+  },
+  getFiltersCount() {
+    const customFilters = this.getFilters();
+    const customFiltersCount = size(omit(customFilters, value => {
+      return isNull(value);
+    }));
+
+    const statesFilter = sortBy(this.getStatesFilters());
+    const defaultStates = this.getDefaultSelectedStates();
```

Comment:
Both need the `sortBy()`?

Reply:
`defaultStates` is sorted at creation

Reply:
That's just how I had it before, is why I'm asking. Unsure if it's needed.

Reply:
so turns out `sortBy` clones, so I'm going to replace the clones in the `get`s